runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java

```diff
@@ -29,10 +29,7 @@
 import java.awt.Point;
 import java.awt.image.BufferedImage;
 import javax.inject.Inject;
-import net.runelite.api.Client;
-import net.runelite.api.InventoryID;
-import net.runelite.api.Item;
-import net.runelite.api.ItemContainer;
+import net.runelite.api.*;
```

Comment:
Don't use star imports. Check the wiki for our code conventions on this.

Reply:
Okay, I think IntelliJ did it automatically. I'll fix both things in a second.